aci/data_source_aci_cloudsvcepg.go

```diff
@@ -0,0 +1,190 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAciCloudServiceEPg() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   dataSourceAciCloudServiceEPgRead,
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"cloud_application_container_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"access_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"azure_private_endpoint": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"custom_service_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"deployment_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"flood_on_encap": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"label_match_criteria": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"preferred_group_member": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"prio": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"cloud_service_epg_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"relation_cloudrs_cloud_epg_ctx": {
+				Type: schema.TypeString,
+
+				Computed:    true,
+				Description: "Query fv:Ctx relationship object",
+			},
+			"relation_fvrs_cons": {
+				Type:        schema.TypeSet,
+				Computed:    true,
+				Description: "Query vzBrCP relationship object",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"prio": {
+							Computed: true,
+							Type:     schema.TypeString,
+						},
+						"target_dn": {
+							Computed: true,
+							Type:     schema.TypeString,
+						},
+					},
+				},
+			},
+			"relation_fvrs_cons_if": {
+				Type:        schema.TypeSet,
+				Computed:    true,
+				Description: "Query vzCPIf relationship object",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"prio": {
+							Computed: true,
+							Type:     schema.TypeString,
+						},
+						"target_dn": {
+							Computed: true,
+							Type:     schema.TypeString,
+						},
+					},
+				},
+			},
+			"relation_fvrs_cust_qos_pol": {
+				Type: schema.TypeString,
+
+				Computed:    true,
+				Description: "Query qos:CustomPol relationship object",
+			},
+			"relation_fvrs_graph_def": {
```

Comment:
The Type of attribute in the docs need to be changed too (for all the attributes that were previously TypeList)